protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -620,7 +501,6 @@ struct AppStorage {
     address ownerCandidate;
     // Well
     mapping(address => bytes) wellOracleSnapshots;
-    uint256 deprecated_beanEthPrice;
     // Silo V3 BDV Migration
     mapping(address => uint256) migratedBdvs;
     // Well/Curve + USD Price Oracle
```

Comment:
can we add `usdTokenPrice` in the natspec?

Reply:
fixed in 7ea86b8f732de1bd4bea2313c197882e15f888b6